administrator/language/en-GB/plg_system_jooa11y.ini

```
@@ -0,0 +1,33 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SYSTEM_JOOA11Y="System - Joomla Accessibility Checker"
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT="Content Container"
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Aria Landmark on the page that will be checked for accessibility. The default setting is <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE="Ignore Regions"
+PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE_DESC="Ignore specific regions within the Content Container. Use commas to seperate classes or elements."
+PLG_SYSTEM_JOOA11Y_FIELD_HEADER_IGNORE="Ignore Headers"
+PLG_SYSTEM_JOOA11Y_FIELD_HEADER_IGNORE_DESC="Ignore specific headings. eg 'h1.jumbotron-heading'. Use commas to seperate classes."
+PLG_SYSTEM_JOOA11Y_FIELD_IMAGE_IGNORE="Ignore Images"
+PLG_SYSTEM_JOOA11Y_FIELD_IMAGE_IGNORE_DESC="Ignore specific images. eg 'spacer.gif'. Use commas to seperate images."
+PLG_SYSTEM_JOOA11Y_FIELD_LINKS_TO_FLAG="Flag Links"
+PLG_SYSTEM_JOOA11Y_FIELD_LINKS_TO_FLAG_DESC="Highlight links you don't want your content editors pointing to (eg development environments, localhost etc)."
+PLG_SYSTEM_JOOA11Y_FIELD_LINK_IGNORE="Ignore Links"
+PLG_SYSTEM_JOOA11Y_FIELD_LINK_IGNORE_DESC="Ignore specific links. eg xxxxx. Use commas to seperate links."
+PLG_SYSTEM_JOOA11Y_FIELD_LINK_SPAN_IGNORE="Ignore Classes within Links"
+PLG_SYSTEM_JOOA11Y_FIELD_LINK_SPAN_IGNORE_DESC="Ignore specific classes within links. Example: &lt;a href='#'&gt;learn more &lt;span class='sr-only-example'&gt;(opens new tab)&lt;/span&gt;&lt;/a&gt;"
+PLG_SYSTEM_JOOA11Y_FIELD_OUTLINE_IGNORE="Outline Panel"
+PLG_SYSTEM_JOOA11Y_FIELD_OUTLINE_IGNORE_DESC="Exclude headings from outline panel."
+PLG_SYSTEM_JOOA11Y_FIELD_PREF_CONTRAST="Contrast Check"
+PLG_SYSTEM_JOOA11Y_FIELD_PREF_DARKMODE="Dark Mode"
+PLG_SYSTEM_JOOA11Y_FIELD_PREF_LABELS="Labels Check"
+PLG_SYSTEM_JOOA11Y_FIELD_PREF_LINKS_ADVANCED="Advanced Links Check"
+PLG_SYSTEM_JOOA11Y_FIELD_PREF_READABILITY="Readability Check"
+PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT="Readability Container"
+PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Aria Landmark on the page that will be checked for readbility. The default setting is <strong>main</strong>."
```

s/readbility/readability

```suggestion
PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Aria Landmark on the page that will be checked for readability. The default setting is <strong>main</strong>."
```